charts/jenkins/README.md

```diff
@@ -13,12 +13,19 @@ This chart will do the following:
 * 1 x Jenkins Master with port 8080 exposed on an external LoadBalancer
 * All using Kubernetes Deployments
 
+## Get Repo Info
+
+```console
+helm repo add jenkinsci https://jenkinsci.github.io/helm-charts
+helm repo update
+```
+
 ## Installing the Chart
 
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install my-release stable/jenkins
+$ helm install my-release jenkinsci/jenkins
```

Comment:
In general, adopt the README simplified template we added for the new prometheus charts. Example: https://github.com/scottrigby/prometheus-helm-charts/tree/main/charts/prometheus

Reply:
I made a PR against this PR branch in your fork. https://github.com/torstenwalter/helm-charts/pull/1